### PR TITLE
[ja] Add a translation in PriorityClass Value Range Description

### PR DESCRIPTION
**Pull request by ADITYADAS1999**
In the documentation of PriorityClass in [english](https://kubernetes.io/docs/concepts/scheduling-eviction/pod-priority-preemption/#priorityclass), the value of a PriorityObject is described as:

> A PriorityClass object can have any 32-bit integer value smaller than or equal to 1 billion. This means that the range of values for a PriorityClass object is from -2147483648 to 1000000000 inclusive

Which is more precisely described about the range of the value but instead in [japanese](https://kubernetes.io/ja/docs/concepts/scheduling-eviction/pod-priority-preemption/#priorityclass), its description is quite simple and missing the mention of negative range.

> PriorityClassオブジェクトは10億以下の任意の32ビットの整数値を持つことができます。

What would you like to be added

Add a translation of the range of the value of PriorityClass.

Why is this needed

This omits the mention of the negative range, which could lead to ambiguity or misunderstandings for readers.

Issue : #42533

### CODE REVIEW

**Comment by sftim** [comment]
/retitle [ja] Fix discrepancy in PriorityClass Value Range Description Between English and Japanese Documentation

File: content/ja/docs/concepts/scheduling-eviction/pod-priority-preemption.md [hunk 1]
```diff
@@ -51,7 +51,8 @@ PriorityClassオブジェクトのメタデータの`name`フィールドにて
 値が大きいほど、高い優先度を示します。
 PriorityClassオブジェクトの名称は[DNSサブドメイン名](/ja/docs/concepts/overview/working-with-objects/names#dns-subdomain-names)として適切であり、かつ`system-`から始まってはいけません。
 
-PriorityClassオブジェクトは10億以下の任意の32ビットの整数値を持つことができます。
+
+PriorityClass オブジェクトには、10 億以下の任意の 32 ビット整数値を含めることができます。これは、PriorityClass オブジェクトの値の範囲が -2147483648 から 1000000000 までであることを意味します。
```
[the remark this answers is not in the document]
Comment (ADITYADAS1999) [reply content/ja/docs/concepts/scheduling-eviction/pod-priority-preemption.md:55]: > You might want to clarify that the number is signed.

Actually, here I want to mention missing negative integer range

**Comment by ADITYADAS1999** [comment]
> In the documentation of PriorityClass in [english](https://kubernetes.io/docs/concepts/scheduling-eviction/pod-priority-preemption/#priorityclass), the value of a PriorityObject is described as:
> 
> > A PriorityClass object can have any 32-bit integer value smaller than or equal to 1 billion. This means that the range of values for a PriorityClass object is from -2147483648 to 1000000000 inclusive
> 
> Which is more precisely described about the range of the value but instead in [japanese](https://kubernetes.io/ja/docs/concepts/scheduling-eviction/pod-priority-preemption/#priorityclass), its description is quite simple and missing the mention of negative range.
> 
> > PriorityClassオブジェクトは10億以下の任意の32ビットの整数値を持つことができます。
> 
> What would you like to be added
> 
> Add a translation of the range of the value of PriorityClass.
> 
> Why is this needed
> 
> This omits the mention of the negative range, which could lead to ambiguity or misunderstandings for readers.
> 
> Issue : #42533

cc :  @kakts @t-inu

File: content/ja/docs/concepts/scheduling-eviction/pod-priority-preemption.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (t-inu) [reply content/ja/docs/concepts/scheduling-eviction/pod-priority-preemption.md]: As a translation of the original text, I don't think there is anything more to explain.

> A PriorityClass object can have any 32-bit integer value smaller than or equal to 1 billion. This means that the range of values for a PriorityClass object is from -2147483648 to 1000000000 inclusive.

File: content/ja/docs/concepts/scheduling-eviction/pod-priority-preemption.md [hunk 2]
```diff
@@ -51,7 +51,8 @@ PriorityClassオブジェクトのメタデータの`name`フィールドにて
 値が大きいほど、高い優先度を示します。
 PriorityClassオブジェクトの名称は[DNSサブドメイン名](/ja/docs/concepts/overview/working-with-objects/names#dns-subdomain-names)として適切であり、かつ`system-`から始まってはいけません。
 
-PriorityClassオブジェクトは10億以下の任意の32ビットの整数値を持つことができます。
+
+PriorityClass オブジェクトには、 1000000000 以下の任意の 32 ビット整数値を含めることができます。これは、PriorityClass オブジェクトの値の範囲が -2147483648 から 1000000000 までであることを意味します。
```
Comment (t-inu) [inline content/ja/docs/concepts/scheduling-eviction/pod-priority-preemption.md:55]:
```suggestion
PriorityClassオブジェクトは10億以下の任意の32ビットの整数値を持つことができます。これは、PriorityClassオブジェクトの値の範囲が-2147483648から1000000000までであることを意味します。
```
- The added blank line is not needed.
- The first sentence need not be changed I think.
- Japanese pages do not include spaces between Japanese and alphanumeric words.

Comment (ADITYADAS1999) [reply content/ja/docs/concepts/scheduling-eviction/pod-priority-preemption.md:55]: Thanks for reviewing @t-inu

**Comment by t-inu** [comment]
/retitle [ja] Add a translation in PriorityClass Value Range Description

**Review by t-inu — COMMENTED** [review]
/lgtm
Thanks for your contribution!

**Comment by ADITYADAS1999** [comment]
> /lgtm Thanks for your contribution!

Thank you 👨🏻‍💻

**Review by nasa9084 — COMMENTED** [review]
/approve
Thank you for your contribution! Great work

**Pull request merged**